drivers/clone.c

```diff
@@ -598,22 +611,54 @@ void upsdrv_updateinfo(void)
 		return;
 	}
 
+	status_init();
+	status_set(ups.status); /* FIXME: Split token words? */
+
+	online = status_get("OL");
+
+	if (!outlet) {
+		/* Outlet was declared OFF (by upstream or assumption) */
+		status_set("OFF");
+		if (ups.load.status && !reported_off) { /* first entrance */
+			reported_off = 1;
+			upslogx(LOG_WARNING, "%s: upstream reports outlet off (setting OFF)",
+				__func__);
```

Comment:
Inform user that the outlet is **reported** off and set `OFF` to make it consistent with `clone-outlet`, the user can see on a first glance the outlet is not giving power.